File: Documentation/Installation_Directory/pxesrv/files/boot.py
#use sys.argv[1], sys.argv[2] etc to retrieve arguments
import sys
print( "#!ipxe")
if (len(sys.argv) > 1):print("echo " + sys.argv[1])
print( "set boot-url http://${dhcp-server}")
print( "kernel ${boot-url}/wimboot")
print( "initrd ${boot-url}/bootmgr                     bootmgr")
print( "initrd ${boot-url}/boot/BCD                    BCD")
print( "initrd ${boot-url}/Boot/boot.sdi               boot.sdi")
print( "initrd ${boot-url}/sources/boot.wim            boot.wim")
print( "boot")

File: Firmware and tools/pxesrv/files/boot.php
<?php
//use $argv[1], $argv[2], etc to retrieve url parameters
echo "#!ipxe\n";
echo "echo ".$argv[1]."\n";
#echo "sleep 5\n";
echo "set boot-url http://${dhcp-server}\n";
echo "kernel ${boot-url}/wimboot\n";
echo "initrd ${boot-url}/bootmgr                     bootmgr\n";
echo "initrd ${boot-url}/boot/BCD                    BCD\n";
echo "initrd ${boot-url}/Boot/boot.sdi               boot.sdi\n";
echo "initrd ${boot-url}/sources/boot.wim            boot.wim\n";
echo "boot\n";
?>  